x-pack/plugins/lens/public/indexpattern_datasource/to_expression.ts

```diff
@@ -76,8 +94,8 @@ function getExpressionForLayer(
           function: 'esaggs',
           arguments: {
             index: [indexPattern.id],
-            metricsAtAllLevels: [false],
-            partialRows: [false],
+            metricsAtAllLevels: [true],
+            partialRows: [true],
```

Comment:
The bug I mentioned in the main thread is caused by setting partialRows to `true`. Instead of getting data.table.tableID.rows = [] when there is not results we're getting a weird object, just like on the screenshot. I am not sure how we can smartly fix it (checking on the level of visualization?) so I'm leaving it to discussion. 

![image](https://user-images.githubusercontent.com/4283304/80579060-b8eb7980-8a09-11ea-8c09-121da3872aad.png)